app/uk/gov/hmrc/gform/views/html/package.scala

```diff
@@ -75,4 +78,18 @@ package object html {
       href = Some(s"/submissions/test-only/payloads/${formTemplate._id.value}/${accessCode.value}"),
       attributes = Map.empty
     )
+
+  private def startNewFormFooterItem(formTemplate: FormTemplate) =
+    new FooterItem(
+      text = Some("Start new form"),
+      href = Some(s"/submissions/new-form/${formTemplate._id.value}"),
```

Comment:
Let's use a reverse route for this link (ie. not hardcoding it as a string literal)